addons/website_event_track_quiz/models/event_quiz.py

```diff
@@ -35,16 +37,22 @@ class QuizQuestion(models.Model):
     quiz_id = fields.Many2one("event.quiz", "Quiz", required=True, ondelete='cascade')
     awarded_points = fields.Integer("Number of Points", compute='_compute_awarded_points')
     answer_ids = fields.One2many('event.quiz.answer', 'question_id', string="Answer")
+    correct_answer = fields.One2many('event.quiz.answer', compute='_compute_correct_answer')
 
     @api.depends('answer_ids.awarded_points')
     def _compute_awarded_points(self):
         for question in self:
             question.awarded_points = sum(question.answer_ids.mapped('awarded_points'))
 
+    @api.depends('answer_ids.is_correct')
+    def _compute_correct_answer(self):
+        for question in self:
+            question.correct_answer = question.answer_ids.filtered(lambda e: e.is_correct)
+
     @api.constrains('answer_ids')
     def _check_answers_integrity(self):
         for question in self:
-            if len(question.answer_ids.filtered(lambda answer: answer.awarded_points)) != 1:
+            if not len(question.correct_answer) == 1:
```

Comment:
Why can't we have several "correct" answers ? Actually is it really required to have "correct" answers ? Sometimes you just have several possible answers with some points. Not sure about the requirement.